drivers/rtc/rtc_ll_stm32.c

```diff
@@ -361,7 +349,7 @@ static int rtc_stm32_set_calibration(const struct device *dev, int32_t calibrati
 	}
 
 	/* wait for recalibration to be ok if a previous recalibration occurred */
-	if (!WAIT_FOR(LL_RTC_IsActiveFlag_RECALP(RTC) == 0, 100000, k_msleep(1))) {
+	if (!WAIT_FOR(LL_RTC_IsActiveFlag_RECALP(RTC) == 0, 10000, k_busy_wait(1000))) {
```

Comment:
Is that still required after the previous commit ?

You mention "k_msleep cause disabling backup domain access by pm", but in previous commit, you add systematic handling of back up domain access. Does the problem remain ?

Note: I've seen this problem as well in other cases and I think we need a back up domain access framework, which would be able to handle such conflicts between devices (not requesting it in this PR, but FYI this is in my TODO list).

Reply:
No, it's not needed with fixed backup access, I removed that change from the commit.